src/setup/index.js

```diff
@@ -39,6 +39,7 @@ export default function () {
             },
             [ONYXKEYS.IS_SIDEBAR_LOADED]: false,
             [ONYXKEYS.SHOULD_SHOW_COMPOSE_INPUT]: true,
+            [ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE]: -1,
```

Comment:
Since -1 is our default value, it makes sense to specify it explicitly. Moreover, this field may need more time to update after starting the application. And it provoked that propTypes drop warn